pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -1226,7 +1235,42 @@ func TestFinallyTaskResultsToPipelineResults_Success(t *testing.T) {
 					}},
 				}},
 			},
-		}},
+		}}, {
+		name: "referencing nonexistent finally task result",
```

Comment:
this description does not match what the test is doing, or am i missing something? 🤔 

```suggestion
		name: "referencing existent finally task result",
```

Reply:
done